koap/npm/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "@juullabs/koap",
+    "version" : "0.3.2-test1",
```

Comment:
Hard coded version as cannot get substitution to work...this used to be simply `"$version"`